docs/docs/headless-cms.md

```diff
@@ -17,8 +17,8 @@ The guides in this section will walk through the process of setting up content s
 
 Other CMS systems you can connect to include:
 
-- [Cosmic JS](https://www.gatsbyjs.org/packages/gatsby-source-cosmicjs)
 - [ButterCMS](https://www.gatsbyjs.org/packages/gatsby-source-buttercms)
+- [Cosmic JS](https://www.gatsbyjs.org/packages/gatsby-source-cosmicjs)
```

Comment:
What's the reason behind re-ordering these entries?

Reply:
Alphabetical? :)

Reply:
There's actually precedent for this! One reason that "Amazon" was chosen as the name for Jeff Bezos' new company was that many link directories at the time, such as Yahoo, were alphabetical. "Abracadabra" was also considered.

Reply:
@calcsam Good call.  Want me to update the list to be alphabetical?

Reply:
No, though thanks for the offer! We're going to order based on what is popular as measured by downloads, as well as CMS size / adoption:

https://github.com/gatsbyjs/gatsby/pull/11254